liquidctl/driver/aquacomputer.py

```diff
@@ -41,9 +48,13 @@
 
 
 class Aquacomputer(UsbHidDriver):
-    # Support for hwmon: aquacomputer_d5next, sensors - 5.15+
+    """Support for hwmon: aquacomputer_d5next driver
+    - D5 Next watercooling pymp: sensors - 5.15+
+    - Farbwerk 360: sensors - 5.18+
+    """
```

Comment:
I think it's better to move this note to the module docstring, or to a simple comment.  It's currently hard for non-Linux users to parse, and as the entire class docstring, it also suggests that this driver is only about hwmon support.

Reply:
Thanks, moved it to the module docstring.